### PR TITLE
update SR-IOV multinetwork doc

### PR DESCRIPTION
**Pull request by zenghui-shi**

### CODE REVIEW

**Comment by ariordan-redhat** [comment]
Edited NIC types in SR-IOV doc.
Added Tech Preview module.

File: modules/nw-multinetwork-sriov.adoc [hunk 1]
```diff
@@ -18,9 +18,9 @@ Two components are required to provide this capability: the SR-IOV network devic
 
 The following Network Interface Card (NIC) models are supported in {product-title}:
 
-* Intel 25G card with vendor ID `0x8086` and device ID `0x158b`
-* Mellanox 25G card with vendor ID `0x15b3` and device ID `0x1015`
-* Mellanox 100G card with vendor ID `0x15b3` and device ID `0x1017`
+* Intel `XXV710-DA2` 25G card with vendor ID `0x8086` and device ID `0x158b`
```
Comment (kalexand-rh) [inline modules/nw-multinetwork-sriov.adoc]: Remove the backticks in the card names.

**Review by kalexand-rh — COMMENTED** [review]
One style change, otherwise this LGTM.

File: modules/nw-multinetwork-sriov.adoc (hunk 1 above)
Comment (ariordan-redhat) [reply modules/nw-multinetwork-sriov.adoc]: Thanks, Kathryn.

**Pull request merged**